### PR TITLE
Add option to expose Kartotherian version T149442

### PR DESCRIPTION
**Pull request by Abbe98**

### CODE REVIEW

**Comment by stephanebisson** [comment]
This is interesting but because we don't deploy kartotherian as an npm module, we don't consistently update the version. You could try including "gitHead" if present. That provides the top git commit hash and it's completely unambiguous.

https://gerrit.wikimedia.org/r/plugins/gitiles/maps/kartotherian/deploy/+/refs/heads/master/node_modules/kartotherian/package.json#137

**Comment by Abbe98** [comment]
@stephanebisson the version is giving a hint and is always available, the commit hash would be even more useful. 👍

File: config.dev.yaml [hunk 1]
```diff
@@ -61,6 +61,8 @@ services:
       #   - x-request-id
       # Uncomment to restrict connections to localhost only
       # interface: localhost
+      # Uncomment to disable the X-Powered-By header
```
Comment (stephanebisson) [inline config.dev.yaml:64]: Where is this flag read from?

Comment (Abbe98) [reply config.dev.yaml:64]: I removed the flag support by accident when cleaning up the git history. It's back in `app.js` now: https://github.com/kartotherian/kartotherian/pull/66/files#diff-0364f57fbff2fabbe941ed20c328ef1aR102

**Pull request merged**